ament_clang_tidy/setup.py

```diff
@@ -0,0 +1,37 @@
+from setuptools import find_packages
+from setuptools import setup
+
+setup(
+    name='ament_clang_tidy',
+    version='6.0.0',
+    packages=find_packages(exclude=['test']),
+    install_requires=['setuptools', 'pyyaml'],
+    package_data={'': [
+        'configuration/.clang-tidy',
+    ]},
+    zip_safe=False,
+    author='John Shepherd',
+    author_email='john@osrfoundation.org',
```

Comment:
Nit: We generally prefer the `@openrobotics.org` email addresses now.  We haven't been consistent on updating the old stuff, but for new stuff we should use that.